Relax the version requirements of some Python modules and remove argparse requirement (is in stdlib)
Author: Samuel Moors (Vrije Universiteit Brussel)
diff -ur CRISPResso2-2.0.44.orig/setup.py CRISPResso2-2.0.44/setup.py
--- CRISPResso2-2.0.44.orig/setup.py	2020-11-14 08:46:36.000000000 +0100
+++ CRISPResso2-2.0.44/setup.py	2020-11-23 18:34:01.477229000 +0100
@@ -88,12 +88,11 @@
               'Programming Language :: Cython',
               ],
           install_requires=[
-              'pandas>=0.15,<=0.24',
-              'matplotlib>=1.3.1,<=2.2.3',
-              'argparse>=1.3,<=1.4',
+              'pandas>=0.15,<0.25',
+              'matplotlib>=1.3.1,<2.3',
     	      'seaborn>0.7.1,<0.10',
-              'jinja2==2.10',
-    	      'scipy==1.1.0',
+              'jinja2>=2.10,<2.11',
+    	      'scipy>=1.1.0,<1.3',
               'numpy>=1.9,<=1.16.6',
               'kiwisolver<1.2',
               'pyparsing<3',
